README.md

```diff
@@ -60,40 +63,48 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖  'Hello Microverse' project <a name="about-project"></a>
 
- 'Hello Microverse' project is the first mini project I've done in microverse
+# 📖 Hello Microverse <a name="about-project"></a>
+
+
+
+**'Hello Microverse project'** is the first mini project I've done in microverse
+
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
+
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="#">HTML & CSS</a></li>
+    <li><a href="#">HTML</a></li>
+  </ul>
+  <ul>
+      <li><a href="#">CSS</a></li>
   </ul>
 </details>
 
-</details>
+
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 
-
 - has **h1** html tag that say hello Microverse
 
 
+
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-
```

Comment:
- [OPTIONAL] Kindly make sure to remove the `LIVE DEMO` because your app is not deployed yet.

Reply:
okay I will take that in my account